src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -994,6 +993,22 @@ def test_sql_db_short_term_retention(self):
                 self.check('retentionDays', '{retention_days_v2}'),
                 self.check('diffBackupIntervalInHours', '{diffbackup_hours_v2}')])
 
+        # Test GET short term retention policy on live database, value equals to v2.
+        self.cmd(
+            'sql db str-policy show -g {resource_group} -s {server_name} -n {database_name}',
+            checks=[
+                self.check('resourceGroup', '{resource_group}'),
+                self.check('retentionDays', '{retention_days_v2}'),
+                self.check('diffBackupIntervalInHours', '{diffbackup_hours_v2}')])
+
+        # Test UPDATE short term retention policy on live database, value updated to v3.
+        self.cmd(
+            'sql db str-policy set -g {resource_group} -s {server_name} -n {database_name} --retention-days {retention_days_v3} --diffbackup-hours {diffbackup_hours_v3}',
```

Comment:
can you remove diffbackup, and verify the diff interval not changed? I.e. stay as v2?

Reply:
Let me call you to explain.